pkg/cli/schemaherokubectlcli/generate.go

```diff
@@ -38,9 +47,9 @@ func GenerateCmd() *cobra.Command {
 		cwd = "."
 	}
 
-	cmd.Flags().String("uri", "", "connection string uri")
-	cmd.Flags().String("driver", "", "name of the database driver to run")
-	cmd.Flags().String("dbname", "", "schemahero database name to write in the yaml")
+	cmd.Flags().String("uri", "", "connection string uri (required)")
+	cmd.Flags().String("driver", "", "name of the database driver to run (required)")
+	cmd.Flags().String("dbname", "", "schemahero database name to write in the yaml (required)")
```

Comment:
I believe there is also https://godoc.org/github.com/spf13/cobra#MarkFlagRequired

Reply:
Yes you're right, I've added it. I did leave the empty string check in there because you can set the flags to empty string and it will still trigger the original panic.